dotnet/docusaurus.config.js

```diff
@@ -41,10 +41,10 @@ module.exports = {
     },
     prism: {
       theme: require('prism-react-renderer/themes/dracula'),
-      additionalLanguages: ['csharp'],
+      additionalLanguages: ['dotnet'],
```

Comment:
Is this expected? This is to support syntax highlighting in C# snippets, I'd expect it to refer to a particular language but I may be wrong.